tests/playbooks/tests_ethtool_coalesce.yml

```diff
@@ -0,0 +1,115 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: testnic1
+    type: veth
+  tasks:
+    - debug:
+        msg: "this is: playbooks/tests_ethtool_.coalesceyml"
+      tags:
+        - always
+
+    - name: "INIT: Ethtool coalesce tests"
+      debug:
+        msg: "##################################################"
+    - include_tasks: tasks/show_interfaces.yml
+    - include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - include_tasks: tasks/assert_device_present.yml
+    - name: Install ethtool (test dependency)
+      package:
+        name: ethtool
+        state: present
+
+    - block:
+        - name: >-
+            TEST: I can create a profile without any coalescing option.
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                type: ethernet
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+        - name: Get profile's coalescing options
+          command: >
+              grep -q ^ETHTOOL_OPTS
+              /etc/sysconfig/network-scripts/ifcfg-{{ interface }}
```

Comment:
Fedora 33 is using key-file by default now. You test should not use hard coded path.
In stead of checking what is written in profile, can you find a way to check whether certain coalesce option is applyed to kernel.

Reply:
I'm afraid not, coalesce settings need kernel support and very few drivers have it. One option could be to use netdevsim but coalesce settings will be supported from Linux 5.11 onwards.

That's why I'm checking the profile instead.

Maybe telling NM to use ifcfg for this test is an option, what do you think?

Reply:
How about use `nmcli -f ethtool  c  show  eth1`?

Reply:
So simple that I did not think about it, I changed the PR to use `nmcli -g ethtool`.
PR updated, thanks!